resources/views/install/config.blade.php

```diff
@@ -56,55 +56,62 @@
             @csrf
             <h3 class="mt-2">Akaun Admin</h3>
             <div class="form-floating mb-3 mt-3">
-                <input type="text" class="form-control" name="adminFullName" id="adminFullname" placeholder="" value="{{ old('adminFullName') }}">
+                <input type="text" class="form-control" name="adminFullName" id="adminFullname" placeholder=""
+                    value="{{ old('adminFullName') }}">
                 <label class="text-dark" for="adminFullName">Nama Penuh</label>
             </div>
             @error('adminFullName')
-                <div class="alert alert-danger">{{ $message }}</div>
+            <div class="alert alert-danger">{{ $message }}</div>
             @enderror
             <div class="form-floating mb-3">
-                <input type="email" class="form-control" name="adminEmailAddress" id="adminEmailAddress" placeholder="" value="{{ old('adminEmailAddress') }}">
+                <input type="email" class="form-control" name="adminEmailAddress" id="adminEmailAddress" placeholder=""
+                    value="{{ old('adminEmailAddress') }}">
                 <label class="text-dark" for="adminEmailAddress">Alamat Emel</label>
             </div>
             @error('adminEmailAddress')
-                <div class="alert alert-danger">{{ $message }}</div>
+            <div class="alert alert-danger">{{ $message }}</div>
             @enderror
             <div class="form-floating mb-3">
-                <input type="password" class="form-control" name="password" id="password" placeholder="" value="{{ old('password') }}">
+                <input type="password" class="form-control" name="password" id="password" placeholder=""
+                    value="{{ old('password') }}">
                 <label class="text-dark" for="password">Kata Laluan</label>
             </div>
             @error('password')
-                <div class="alert alert-danger">{{ $message }}</div>
+            <div class="alert alert-danger">{{ $message }}</div>
             @enderror
             <div class="form-floating mb-3">
-                <input type="password" class="form-control" name="password_confirmation" id="password_confirmation" placeholder="">
+                <input type="password" class="form-control" name="password_confirmation" id="password_confirmation"
+                    placeholder="">
                 <label class="text-dark" for="password_confirmation">Sahkan Kata Laluan</label>
             </div>
-            <h3 class="mt-2">Tetapan Sistem</h3>
+            <h3 class="mt-2">Nama Sistem Anda?</h3>
```

Comment:
Maybe instead of change the header, leave it and then add a label for the name field.

Reply:
Up to you. No harm :)

Reply:
I think it's better tho. Since well, it's the section for editing the system settings. In the section, there's a setting for name, language and logo.